fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/FabricItemSettings.java

```diff
@@ -42,6 +42,14 @@ public FabricItemSettings equipmentSlot(EquipmentSlotProvider equipmentSlotProvi
 		return this;
 	}
 
+	/**
+	 * Sets the custom damage handler of the item.
```

Comment:
Really a javadoc nitpick, but possibly add `@see CustomDamageHandler` so people can see it relates to the handler.